src/jit/codegeninterface.h

```diff
@@ -296,59 +276,14 @@ class CodeGenInterface
 #endif
 
 public:
-#if FEATURE_STACK_FP_X87
-    FlatFPStateX87 compCurFPState;
-    unsigned       genFPregCnt; // count of current FP reg. vars (including dead but unpopped ones)
-
-    void SetRegVarFloat(regNumber reg, var_types type, LclVarDsc* varDsc);
-
-    void inst_FN(instruction ins, unsigned stk);
-
-    //  Keeps track of the current level of the FP coprocessor stack
-    //  (excluding FP reg. vars).
-    //  Do not use directly, instead use the processor agnostic accessor
-    //  methods below
-    //
-    unsigned genFPstkLevel;
-
-    void genResetFPstkLevel(unsigned newValue = 0);
-    unsigned        genGetFPstkLevel();
-    FlatFPStateX87* FlatFPAllocFPState(FlatFPStateX87* pInitFrom = 0);
-
-    void genIncrementFPstkLevel(unsigned inc = 1);
-    void genDecrementFPstkLevel(unsigned dec = 1);
-
-    static const char* regVarNameStackFP(regNumber reg);
-
-    // FlatFPStateX87_ functions are the actual verbs to do stuff
-    // like doing a transition, loading   register, etc. It's also
-    // responsible for emitting the x87 code to do so. We keep
-    // them in Compiler because we don't want to store a pointer to the
-    // emitter.
-    void FlatFPX87_MoveToTOS(FlatFPStateX87* pState, unsigned iVirtual, bool bEmitCode = true);
-    void FlatFPX87_SwapStack(FlatFPStateX87* pState, unsigned i, unsigned j, bool bEmitCode = true);
-
-#endif // FEATURE_STACK_FP_X87
-
-#ifndef LEGACY_BACKEND
     regNumber genGetAssignedReg(GenTree* tree);
```

Comment:
It appears that this is unused.

Reply:
Removed. There will probably be a bit of fallout like this.